default.nix

```diff
@@ -19,30 +19,29 @@
 # Please run `nix/regenerate.sh` after modifying stack.yaml
 # or relevant part of cabal configuration files.
 # When switching to recent stackage or hackage package version,
-# you might also need to update the iohk-nix common lib. You 
+# you might also need to update the iohk-nix common lib. You
 # can do so by running the `nix/update-iohk-nix.sh` script.
-# 
+#
 # More information about iohk-nix and nix-tools is available at:
 # https://github.com/input-output-hk/iohk-nix/blob/master/docs/nix-toolification.org#for-a-stackage-project
 #
 
-
-# We will need to import the iohk-nix common lib, which includes
-# the nix-tools tooling.
 let
   commonLib = import ./nix/iohk-common.nix;
-in
-# This file needs to export a function that takes
-# the arguments it is passed and forwards them to
-# the default-nix template from iohk-nix. This is
-# important so that the release.nix file can properly
-# parameterize this file when targetting different
-# hosts.
-{ ... }@args:
-# We will instantiate the default-nix template with the
-# nix/pkgs.nix file...
-commonLib.nix-tools.default-nix ./nix/pkgs.nix args 
-# ... and add additional non-haskell packages we want to build on CI:
-// {
-  
+
+  # Haskell Packages produced by nix-tools
+  haskellPackages = import ./nix/haskell-packages.nix {};
+  # TODO: remove when proxy and old cardano configs no
+  # longer required.
+  oldCardanoSrc = import ./nix/old-cardano.nix {
```

Comment:
Why is it called "old" cardano?

Reply:
to distinguish from this repo being "new" cardano. temporary until we no longer need the "old cardano" as a proxy.